data_visualization_in_seaborn/data_visualization_seaborn.md

```diff
@@ -0,0 +1,684 @@
+sns.set_context("poster")
+
+sns.relplot(data = covid_data,
+            x="Age", y="Glucose",
+            hue="Classification", style = "Classification")
+```
+![Age and glucose scatterplot from the previous figure, with much larger font for axis scales and labels, and larger markers for the points in the plot.](media/seaborn_scatter_7.png)
+
+```Python
+# set context back to notebook (the default)
+sns.set_context("notebook")
+```
+
+### Quiz: Scatterplots
+
+What is the command to create a scatterplot in seaborn?
+
+[[sns.relplot()]]
+<script>
+  let input = "@input".trim();
+  /relplot/.test(input) || /regplot/.test(input);
+</script>
+***
+<div class = "answer">
+By default, the `relplot` command creates a scatterplot (although it can also be used to create [line plots](#line-plots)). You can also use `regplot` to create scatterplots, as we'll see in the [trend lines](#trend-lines) section.
+
+Note the `sns` before the function name. That's because we imported the seaborn library with an alias, as is conventional. See [making plots in seaborn](#making-plots-in-seaborn).
+</div>
+***
+
+True or False: The only two crucial arguments to supply (in addition to providing the data) for a scatterplot are x and y.
+
+[(X)] TRUE
+[( )] FALSE
+***
+<div class = "answer">
+While x and y are the only two **crucial** arguments for you to supply the plotting function, you may want to include others, such as hue and style, to communicate information about additional variables in the data.
+</div>
+***
+
+## Histograms
+
+Histograms show the distribution of a continuous variable. The values of the variable are shown along the x-axis, and data are grouped into bins, with the height of each bin corresponding to the number of data points in that bin. In other words, it communicates where your data for a given variable fall within its range. It is a great way to quickly assess for symmetry vs skew, outliers, and less common issues like multimodality.
+
+We'll continue using the same data we explored to make scatter plots.
+
+### Basic histogram
+
+Because histograms show just one variable, the only aesthetic they require is x. The y-axis of the plot will just show the counts of observations in each bin on the x-axis. (Note that it is possible to provide `sns.displot` with y instead of x, in which case it will generate a sideways histogram.)
+
+```python
+sns.displot(covid_data, x="val_age")
+```
+
+![Histogram showing the distribution of age values.](media/seaborn_hist_1.png)
+
+
+### Change the number of bins
+
+The appearance of a histogram can change a lot depending on the number of bins you use along the x-axis. It's a good idea to try a few different sets of bins to see what works well for communicating this distribution.
+
+```python
+sns.displot(covid_data, x="val_age", binwidth=1)
+
+```
+
+![Histogram showing the distribution of ages with many narrow bins, each representing a single year.](media/seaborn_hist_2.png)
```

Comment:
Added to main text rather than alt text, since I thought it was a suggestion that everybody would benefit from, including folks with visual access to the plots :)